pkgs/applications/graphics/tauray/default.nix

```diff
@@ -0,0 +1,65 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, graphviz
+, doxygen
+, pkg-config
+, python3
+, libcbor
+, czmq
+, glm
+, nng
+, mbedtls
+, openxr-loader
+, SDL2
+, shaderc
+, spirv-headers
+, vulkan-headers
+, vulkan-loader
+, wayland
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "tauray";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "vga-group";
+    repo = "tauray";
+    rev = "v${finalAttrs.version}";
+    sha256 = "sha256-Q3Dv96iptDvmBp9mIVSYzb7EnIzD1kFejwrFuc1HqdU=";
+    fetchSubmodules = true;
+  };
+
+  buildInputs = [
+    czmq
+    glm
+    libcbor
+    nng
+    mbedtls
+    openxr-loader
+    (SDL2.override { withStatic = true; })
```

Comment:
is this necessary?

Reply:
The build scripts don't seem to find SDL2::SDLmain otherwise.